### PR TITLE
First pass at relase docs

### PR DESCRIPTION
**Pull request by hardillb**
Outline the basic tasks needed for a release.

### CODE REVIEW

**Comment by hardillb** [comment]
@knolleary I have written this with an opinion on version numbering, in the meeting this morning you said you would raise an issue about this, so feel free to request changes to suite.

File: development/release.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Release Process
+
+The follow process should be followed for a FlowForge release
+
+## Setup
+
+ - Decide who will be Release Manager for this release. This should be shared across the whole team to prevent it becoming a single point of failure.
+ - Create a Release check list issue (using template) on the flowforge/admin project to keep track of all components included in the release (e.g. https://github.com/flowforge/admin/issues/7)
```
Comment (knolleary) [inline development/release.md:8]:
```suggestion
 - Create a Release check list issue (using template) on the flowforge/admin project to keep track of all components included in the release (e.g. https://github.com/flowforge/admin/issues/7)
 - Assign the issue to the Release Manager
```

File: development/packaging.md [hunk 2]
```diff
@@ -75,3 +76,8 @@ The `package.json` must contain the following keys
     },
     ```
  
+### Package Version Numbering
+
+Package numbers should follow the Semantic Versioning Scheme as laid out on [semver.org](https://semver.org/).
+
+Each component will maintain it's own numbering regardless of the core FlowForge project and only update it's version as needed.
```
Comment (knolleary) [inline development/packaging.md:83]: For the 'internal' components of the platform, I think we should say they need to keep their SemVer `major.minor` number in sync, but can have individual `fix` level releases.

Comment (hardillb) [reply development/packaging.md]: Updated to cover this

**Pull request merged**